starlark/experimental/shiftleft.star

```diff
@@ -0,0 +1,49 @@
+# vi:syntax=python
+
+load("/starlark/stable/pipeline", "storage_resource")
+
+__doc__ = """
+# Shiftleft
+
+Provides methods for running [shiftleft scans](https://www.shiftleft.io/scan/) on your CI.
+
+To import, add the following to your Dispatchfile:
+
+```
+load("github.com/mesosphere/dispatch-catalog/starlark/experimental/shiftleft@master", "sast_scan")
+```
+
+"""
+
+def sast_scan(git, task_name, image_and_tag="shiftleft/sast-scan:latest", src=None, extra_scan_options=None, **kwargs):
```

Comment:
```suggestion
def sast_scan(task_name, git_name, image="shiftleft/sast-scan:latest", scan_options=[], **kwargs):
```
I'm also wondering if we should just always use the latest image, and if we need to provide `**kwargs`.

Reply:
Right now, there are no tags except the latest image - https://hub.docker.com/r/shiftleft/sast-scan/tags

as for the keyword args, it could be useful for sending envvars (log level, token etc..)

Reply:
What I mean is that we can consider not allowing people to pass in custom image.

If you think it's better to allow a custom image, I'd recommend to use the same parameter name as in `go` and `go_test`.

Reply:
I mean considering that they don't have any other images, i think it can come in handy to allow for image parameter. 

renamed the param to be consitent with other catalog tasks